content/shared/v3-core-plugins/_index.md

```diff
@@ -57,70 +72,86 @@ If you're running multiple {{% product-name %}} instances (distributed deploymen
+- [Create a custom plugin](#create-a-custom-plugin) - For custom functionality
 
-InfluxData maintains a repository of contributed plugins that you can use as-is or as a starting point for your own plugin.
+### Use example plugins
 
-#### From local files
+The InfluxData team maintains a repository of example plugins you can use immediately:
 
-You can copy example plugins from the [influxdb3_plugins repository](https://github.com/influxdata/influxdb3_plugins) to your local plugin directory:
+1. **Browse available plugins**: Visit the [influxdb3_plugins repository](https://github.com/influxdata/influxdb3_plugins) to find examples for:
+   - **Data transformation**: Process and transform incoming data
+   - **Alerting**: Send notifications based on data thresholds
+   - **Aggregation**: Calculate statistics on time series data
+   - **Integration**: Connect to external services and APIs
+   - **System monitoring**: Track resource usage and health metrics
 
+2. **Choose how to access plugins**:
+
+**Option A: Copy plugins to your local directory**
+   
 ```bash
 # Clone the repository
 git clone https://github.com/influxdata/influxdb3_plugins.git
-
-# Copy example plugins to your plugin directory
-cp -r influxdb3_plugins/examples/wal_plugin/* /path/to/plugins/
+   
+# Copy a plugin to your configured plugin directory
+cp influxdb3_plugins/examples/schedule/system_metrics/system_metrics.py /path/to/plugins/
 ```
 
-#### Directly from GitHub
+**Option B: Use plugins directly from GitHub**
 
 You can use plugins directly from GitHub without downloading them first by using the `gh:` prefix in the plugin filename:
-
+    
 ```bash
-# Use a plugin directly from GitHub
 influxdb3 create trigger \
     --trigger-spec "every:1m" \
     --plugin-filename "gh:examples/schedule/system_metrics/system_metrics.py" \
     --database my_database \
     system_metrics
 ```
 
-> [!Note]
-> #### Find and contribute plugins
->
-> The plugins repository includes examples for various use cases:
-> 
-> - **Data transformation**: Process and transform incoming data
-> - **Alerting**: Send notifications based on data thresholds
-> - **Aggregation**: Calculate statistics on time series data
-> - **Integration**: Connect to external services and APIs
-> - **System monitoring**: Track resource usage and health metrics
->
-> Visit [influxdata/influxdb3_plugins](https://github.com/influxdata/influxdb3_plugins)
-> to browse available plugins or contribute your own.
+Plugins have various functions such as: 
+
+- Receive plugin-specific arguments (such as written data, call time, or an HTTP request)
+- Can receive keyword arguments (as `args`) from _trigger arguments_
+- Can access the `influxdb3_local` shared API for writing, querying, and managing state
+
+### Create a custom plugin
+
+When you need custom functionality, you can create your own plugin by doing the following:
 
-### Create a plugin
+#### Step 1: Choose your plugin type
```

Comment:
We typically avoid using numbered headings and instead insert an ordered TOC before the steps.